src/main/webapp/theme/utilities/utilities.scss

```diff
@@ -0,0 +1,14 @@
+@import "./generator.scss";
```

Comment:
Nice one, but I guess a bit overkill for such a small project. It generates a huge amount of unused CSS.

Reply:
I usually like to do things the right way from the beginning. Like having global classes defined to reduce code duplicate/ different naming and so on. That's why I created those functions that are generating the styling. When I need new properties, I will add them to the variables.scss file.